### DISCUSSION TITLE
When to meet as the front-end team

### DISCUSSION THREAD

**Discussion by meminciftci** [bounswe/bounswe2024group11#163] [Polls]
We have divided into sub-teams at our last meeting. As the front-end team we need to hold a meeting as soon as possible and start our work. Today (18 April, Thursday), let us decide on a time slot to discuss our initial and next steps.

Please choose the hours you are available for today's meeting. 

@meminciftci @umitcan07 @tenacke @hks1444 @ArdaVural

**Comment by meminciftci** [comment]
If you have more than one preferred time-slot, you can leave a comment.

**Comment by umitcan07** [comment]
I have an exam in between 17-18.

**Reply by umitcan07** [reply]
Other 3 sounds better

**Comment by hks1444** [comment]
All of them are okay for me. But I would prefer to make this meeting early.

**Comment by ArdaVural** [comment]
I prefer 19.00 but 20.00 is also okay for me. (I assume earlier than 19.00 is not available because of Umitcan's exam.)

**Comment by tenacke** [comment]
I am not available after 19. It is better to meet earlier

**Comment by meminciftci** [comment]
So the most preferred time slot is 18:00-18:59. 

Is it OK for you to attend the meeting right after your exam @umitcan07 ? Or we can start the meeting 18:30. Also it will probably a short one since we have not done anything yet to discuss about.

**Comment by umitcan07** [comment]
sounds good

Muhammet Emin Çiftçi ***@***.***>, 18 Nis 2024 Per, 13:31
tarihinde şunu yazdı:
